src/firefly/java/edu/caltech/ipac/firefly/server/LocalFSWorkspace.java

```diff
@@ -79,6 +80,11 @@ public WsResponse getList(String parentUri, int depth) throws IOException {
     }
 
     @Override
+    public WsResponse getFile(String fileUri, Consumer consumer) throws WsException {
+        throw new IllegalArgumentException("not implemented");
+    }
```

Comment:
Thanks for the review. Yes, I've left out the parameter type to consume because i thought it could be anything we want, not only inputstream. 
I will merge and we can add the implementation and specifying the generic type at the same time.